lib/simp/cli/commands/command_family.rb

```diff
@@ -0,0 +1,83 @@
+require 'simp/cli/commands/command'
+
+# This class is the API for a Command Family.
+class Simp::Cli::Commands::CommandFamily < Simp::Cli::Commands::Command
+  # @retrun [String] "snake-case" name of command
+  def snakecase_name
+    self.class.to_s.split('::').last.gsub(%r{(?<!^)[A-Z]}) { "_#{$&}" }.downcase
+  end
+
+  def sub_commands
+    return @sub_commands if @sub_commands
+    @sub_commands = {}
+    subcmd_files = Dir.glob(File.expand_path("#{snakecase_name}/*.rb", __dir__)).sort_by(&:to_s)
+    subcmd_files.each { |file| require file }
+    Simp::Cli::Commands::Environment.constants.each do |constant|
```

Comment:
If this is supposed to be a generic command family, it shouldn't be grabbing constants from `Simp::Cli::Commands::Environment`

Reply:
I'll fix it.